Time::getContext() limited to returning U8

| | |
|:---|:---|
|**_F´ Version_**| devel |
|**_Affected Component_**| Fw::Time  |
---
## Problem Description

The Time class declares getContext() as returning FwTimeContextStoreType in the [hpp](https://github.com/nasa/fprime/blob/8db8e4f22255eb5c9359043dab70f9442a63f82c/Fw/Time/Time.hpp#L33) , but defines getContext() as returning U8 in the [cpp](https://github.com/nasa/fprime/blob/8db8e4f22255eb5c9359043dab70f9442a63f82c/Fw/Time/Time.cpp#L146) .

This limits the definition of FwTimeContextStoreType to U8. We'd like to use the Context to store a boot count, but 0-254 is a bit constraining in terms of number of boots. 

## How to Reproduce

1. Define FwTimeContextStoreType as something other than U8
2. Build a Topology

I got

```
/home/abcouwer/Dev/fprime-fwca/Fw/Time/Time.cpp:146:8: error: no declaration matches ‘U8 Fw::Time::getContext() const’
  146 |     U8 Time::getContext(void) const {
      |        ^~~~
In file included from /home/abcouwer/Dev/fprime-fwca/Fw/Time/Time.cpp:1:
/home/abcouwer/Dev/fprime-fwca/Fw/Time/Time.hpp:33:36: note: candidate is: ‘U16 Fw::Time::getContext() const’
   33 |             FwTimeContextStoreType getContext(void) const; // !< get the context value
      |                                    ^~~~~~~~~~
In file included from /home/abcouwer/Dev/fprime-fwca/Fw/Time/Time.cpp:1:
/home/abcouwer/Dev/fprime-fwca/Fw/Time/Time.hpp:10:11: note: ‘class Fw::Time’ defined here
   10 |     class Time: public Serializable {
      |           ^~~~
```


## Expected Behavior

Would be nice to have flexibility here. If it has never been tweaked before, I forsee issues with GDS or other parts of the framework. I haven't tried changing the Fw yet and seeing how that goes in Topology and GDS. I can update this issue and PR if I make one.

--- a/Fw/Time/Time.cpp
+++ b/Fw/Time/Time.cpp
@@ -144,7 +144,7 @@ namespace Fw {
         return this->m_timeBase;
     }
 
-    U8 Time::getContext() const {
+    FwTimeContextStoreType Time::getContext() const {
         return this->m_timeContext;
     }
 

